data_transaction_openapidoc_proposal.yml

```diff
@@ -0,0 +1,727 @@
+components:
```

Comment:
I think this file is useless in the code base, maybe it can be annexed in the issue of the proposal

Reply:
Yeah. It was just for the initial proposal.

Reply:
removed